packages/blade/src/components/ListView/_decisions/decisions.md

```diff
@@ -0,0 +1,481 @@
+          onClick={({ name, value }) => setDuration(name)}
+        />
+        <ActionListItem
+          title="1 week"
+          value="1w"
+          onClick={({ name, value }) => setDuration(name)}
+        />
+        <ActionListItem
+          title="1 month"
+          value="1m"
+          onClick={({ name, value }) => setDuration(name)}
+        />
+      </ActionList>
+    </DropdownOverlay>
+  </Dropdown>
+
+  <Menu>
+    <FilterChip value={utrNumber} onClearButtonClick={({ value }) => setUtrNumber(undefined)}>
+      UTR Number
+    </FilterChip>
+    <MenuOverlay>
+      <TextInput ref={utrNumberInputValueRef} />
+      <Button onClick={() => setUtrNumber(utrNumberInputValueRef.current.value)}>Submit</Button>
+    </MenuOverlay>
+  </Menu>
+
+  <Dropdown>
+    <FilterChip
+      value={status}
+      defaultValue="Initiated"
+      onClearButtonClick={({ value }) => setStatus(undefined)}
+    >
+      Status
+    </FilterChip>
+    <DropdownOverlay>
+      <ActionList>
+        <ActionListItem title="All" onClick={({ name, value }) => setStatus(name)} />
+        <ActionListItem title="Pending" onClick={({ name, value }) => setStatus(name)} />
+        <ActionListItem title="Completed" onClick={({ name, value }) => setStatus(name)} />
+      </ActionList>
+    </DropdownOverlay>
+  </Dropdown>
+</FilterChipGroup>;
+```
+
+#### Props
+
+##### FilterChipGroup
+
+```ts
+type FilterChipGroupProps = {
+  /**
+   * Children prop. Supports Dropdown, Menu components
+   *
+   */
+  children: React.ReactNode;
+};
+```
+
+##### FilterChip
+
+FilterChip can be used as a trigger for Dropdown and Menu
+
+```ts
+type FilterChipProps = {
+  /**
+   * Controlled value of FilterChip.
+   *
+   * FilterChip is always a controlled component since selected state of it comes from other components like Menu, Dropdown.
+   */
+  value: string;
+
+  /**
+   * Change handler when FilterChip's value is cleared
+   */
+  onClearButtonClick: ({ value }: { value: string }) => void;
+
+  /**
+   * Children. Title of the Chip
+   */
+  children: string;
+
+  /**
+   * Disabled state for Chip
+   */
+  isDisabled?: boolean;
+};
+```
+
+### ListView (Layout Component)
+
+Layout Component for handling the overall layout of ListView and Filters. More about layout components at [pattern-terminology.md](./pattern-terminology.md)
+
+```jsx
+<ListView>
+  <ListViewFilters
+    quickFilters={[
+      {
+        title: 'All',
+        onClick: () => {},
+      },
+      {
+        title: 'Pending',
+        onClick: () => {},
+      },
+    ]}
+    onSearch={({ value, searchType }) => {
+      /* filter on search */
+    }}
+  >
+    <FilterGroup />
+  </ListViewFilters>
+  <Table />
+</ListView>
+```
+
+#### Props
+
+##### ListView
+
+```ts
+type ListViewProps = {
+  children: React.ReactNode;
+};
+```
+
+##### ListViewFilters
+
+<img src="image-4.png" width="600px" />
+
+```ts
+type QuickFilter = {
+  title: string;
+  onClick: (e: React.MouseEvent) => void;
```

Comment:
We will also need onChange, value & controlled/uncontrolled state for QuickFilters. 

Because QuickFilters will have the same semantics of RadioGroup it can also change on keyboard events not just click. 

Plus this will also be a requirement if we want this to work independently as i've mentioned [here](https://github.com/razorpay/blade/pull/2453/files#r1912960927)